plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -369,6 +369,13 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
                 )
             }
 
+            is CodeModernizerStartJobResult.UnabletoStartOrUploadJob -> {
+                CodeModernizerJobCompletedResult.UnableToCreateJob(
+                    message("codemodernizer.notification.warn.unable_to_start_job_due_to_upload_failure", result.url),
```

Comment:
Around here: https://github.com/aws/aws-toolkit-jetbrains/blob/fb1f9d2ae20cde7a52c499efdb01684502c3f53f/plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt#L554

You should be doing something like https://github.com/aws/aws-toolkit-jetbrains/blob/fb1f9d2ae20cde7a52c499efdb01684502c3f53f/plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt#L598

to display a specific notification with the appropriate URL.

Reply:
I am sending notification to the user if the upload fails, I am trying to show in the Q chat window instead as WeaverBird does the same but If the UX needs to do notification then I can go ahead with this approach.

Reply:
Tested this in other place and this works in the chat.
![image](https://github.com/aws/aws-toolkit-jetbrains/assets/141967714/e2e9dfca-a683-46cc-878f-546b8e06a2fb)